apps/src/lib/levelbuilder/code-docs-editor/ExampleEditor.jsx

```diff
@@ -0,0 +1,101 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import color from '@cdo/apps/util/color';
+import HelpTip from '@cdo/apps/lib/ui/HelpTip';
+import TextareaWithMarkdownPreview from '@cdo/apps/lib/levelbuilder/TextareaWithMarkdownPreview';
+
+const APP_DISPLAY_OPTIONS = {
+  directly: 'Embed app with code directly',
+  withCode: 'Display app with code from code field above'
```

Comment:
do you have existing examples of these two display styles on https://studio.code.org/docs that you can link to? or is this new functionality we are adding as part of the move?

Reply:
It is an existing setting on curriculum builder. https://www.codecurricula.com/docs/spritelab/gamelab_jumpTo/ uses "embed code directly". https://www.codecurricula.com/docs/gamelab/camera.y/ uses "Display app with code from code field above"